Makefile

```diff
@@ -8,6 +8,9 @@ FLAGS=-X main.Version=$(VERSION) -s -w
 CERT="Developer ID Application: 99designs Inc (NRM9HVJ62Z)"
 SRC=$(shell find . -name '*.go')
 
+test:
+	go test $(shell go list ./... | grep -v /vendor/)
```

Comment:
ah, of course, i saw the use of glide and just assumed from there. travis != our buildkite setup